src/lib/containers/table/table-top/ProgrammaticOptions.tsx

```diff
@@ -56,39 +63,91 @@ function ProgrammaticOptions({
     getData()
   }, [queryBundleRequest, queryResultBundle])
 
+  const installationInstructions = <p>Installation instructions are available at our
+    <a
+      className="ProgrammaticOptions__docslink"
+      href="https://help.synapse.org/docs/Installing-Synapse-API-Clients.1985249668.html"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      {' '}
+      Synapse API Documentation Site
```

Comment:
I am curious why you want to add a space character inside the anchor tag instead of outside?

Reply:
Good question!  This space character was in the code previously (see the old line 84), and I didn't clean it up :) I'll take care of it, thanks @emmanmills !

Reply:
Pulled out the space character, and verified the output UI looked ok.